bin/wiki_entity_linking/wikidata_pretrain_kb.py

```diff
@@ -13,63 +13,73 @@
-    wp_xml=("Path to the downloaded Wikipedia XML dump.", "positional", None, Path),
+    wikidata_json=("Path to the downloaded WikiData JSON dump.", "positional", None, Path),
+    wikipedia_xml=("Path to the downloaded Wikipedia XML dump.", "positional", None, Path),
     output_dir=("Output directory", "positional", None, Path),
     model=("Model name, should include pretrained vectors.", "positional", None, str),
+    model_is_path=("Model is a path not a model name.", "flag", "mp"),
     max_per_alias=("Max. # entities per alias (default 10)", "option", "a", int),
     min_freq=("Min. count of an entity in the corpus (default 20)", "option", "f", int),
     min_pair=("Min. count of entity-alias pairs (default 5)", "option", "c", int),
     entity_vector_length=("Length of entity vectors (default 64)", "option", "v", int),
-    loc_prior_prob=("Location to file with prior probabilities", "option", "p", Path),
-    loc_entity_defs=("Location to file with entity definitions", "option", "d", Path),
-    loc_entity_desc=("Location to file with entity descriptions", "option", "s", Path),
     limit=("Optional threshold to limit lines read from dumps", "option", "l", int),
+    create_prior_probs=("Flag to recreate prior probs", "flag", "cp"),
+    create_training_set=("Flag to recreate training data", "flag", "t"),
+    lang=("Optional language for which to get wikidata titles. Defaults to 'en'", "option", "la", str),
 )
 def main(
-    wd_json,
-    wp_xml,
+    wikidata_json,
+    wikipedia_xml,
     output_dir,
     model,
+    model_is_path=False,
     max_per_alias=10,
     min_freq=20,
     min_pair=5,
     entity_vector_length=64,
-    loc_prior_prob=None,
-    loc_entity_defs=None,
-    loc_entity_desc=None,
     limit=None,
+    create_prior_probs=False,
+    create_training_set=False,
+    lang="en",
 ):
-    print(now(), "Creating KB with Wikipedia and WikiData")
-    print()
+
+    entity_defs_path = output_dir / "entity_defs.csv"
+    entity_descr_path = output_dir / "entity_descriptions.csv"
+    entity_freq_path = output_dir / "entity_freq.csv"
+    prior_prob_path = output_dir / "prior_prob.csv"
+    training_entities_path = output_dir / TRAINING_DATA_FILE
+    kb_path = output_dir / KB_FILE
+
+    logger.info("Creating KB with Wikipedia and WikiData")
 
     if limit is not None:
-        print("Warning: reading only", limit, "lines of Wikipedia/Wikidata dumps.")
+        logger.warning("Warning: reading only {} lines of Wikipedia/Wikidata dumps.".format(limit))
 
     # STEP 0: set up IO
     if not output_dir.exists():
-        output_dir.mkdir()
+        output_dir.mkdir(parents=True)
 
     # STEP 1: create the NLP object
-    print(now(), "STEP 1: loaded model", model)
-    nlp = spacy.load(model)
+    logger.info("STEP 1: loading model {}".format(model))
+    if model_is_path:
+        nlp = spacy.load(Path(model))
+    else:
+        nlp = spacy.load(model)
 
```

Comment:
Are you sure this change is necessary? `spacy.load(model)` should work with a package name like `en_core_web_lg`, a string denoting a path, or an actual `Path`

Reply:
You're right! I'll remove it and update the docstring.